utilities.js

```diff
@@ -1,13 +1,4 @@
-export function streamToIterable(stream) {
-	return {
-		[Symbol.asyncIterator]: stream[Symbol.asyncIterator].bind(stream),
-	};
-}
-
-export async function * combineAsyncIterables(iterable1, iterable2) {
-	const iterator1 = iterable1[Symbol.asyncIterator]();
-	const iterator2 = iterable2[Symbol.asyncIterator]();
```

Comment:
Node.js streams are both iterables and iterators.

Reply:
The function name is no longer correct then.

Reply:
Fixed. :+1: